CHANGELOG.md

```diff
@@ -1,3 +1,132 @@
+- Introduce resource quota for Alibaba provider [#6458](https://github.com/kubermatic/kubermatic/issues/6458) 
+- Add Gatekeeper health to cluster health status [#6461](https://github.com/kubermatic/kubermatic/issues/6461) 
+- Remove CoreOS Support [#6465](https://github.com/kubermatic/kubermatic/issues/6465) 
+- Relax memory limit for openvpn container [#6467](https://github.com/kubermatic/kubermatic/issues/6467) 
+- Enable VM resource quota for AWS provider [#6468](https://github.com/kubermatic/kubermatic/issues/6468) 
+- Multus has been added as an addon.  [#6477](https://github.com/kubermatic/kubermatic/issues/6477) 
+- CoreDNS version now based on Kubernetes version [#6501](https://github.com/kubermatic/kubermatic/issues/6501) 
+- Add support for &#34;use-octavia&#34; setting in Openstack provider specs. It defaults to &#34;true&#34; but leaves the possibility to set it to &#34;false&#34; if your provider doesn&#39;t support Octavia yet but Neutron LBaaSv2 [#6529](https://github.com/kubermatic/kubermatic/issues/6529) 
+- Add components override field to set nodeportrange for apiserver [#6533](https://github.com/kubermatic/kubermatic/issues/6533) 
+- Extend Kubermatic Constraint Template spec with a selector which enables cluster filtering for EE Kubermatic.  [#6534](https://github.com/kubermatic/kubermatic/issues/6534) 
+- CCM/CSI migration [#6535](https://github.com/kubermatic/kubermatic/issues/6535) 
+- Overhaul CA handling, allow to configure a global CA bundle for every component. The OIDC CA ile has been removed, manual updates can be necessary. [#6538](https://github.com/kubermatic/kubermatic/issues/6538) 
+- OpenShift support is removed.  [#6539](https://github.com/kubermatic/kubermatic/issues/6539) 
+- Fix issue with gatekeeper not recognizing the AdmissionReview v1 version by changing the webhook to use v1beta1 [#6550](https://github.com/kubermatic/kubermatic/issues/6550) 
+- OpenStack: Add support for &#34;use-octavia&#34; setting in Cluster Openstack cloud specs. [#6565](https://github.com/kubermatic/kubermatic/issues/6565) 
+- Kubermatic Constraint Templates now support cluster filtering to allow users to manage the CT&#39;s and policies across different clusters more easily. The feature is available for Kubermatic EE.  [#6567](https://github.com/kubermatic/kubermatic/issues/6567) 
+- Fix KAS service port in Tunneling agent configuration. [#6569](https://github.com/kubermatic/kubermatic/issues/6569) 
+- Add support for Hetzner CCM [#6588](https://github.com/kubermatic/kubermatic/issues/6588) 
+- Increase default gatekeeper webhook timeout from 2 to 10 seconds, and added option in cluster settings to configure it. [#6590](https://github.com/kubermatic/kubermatic/issues/6590) 
+- Add support in Openstack datacenters to explicitly enable certain flavor types. [#6612](https://github.com/kubermatic/kubermatic/issues/6612) 
+- Extend SA controller for new SA type. [#6634](https://github.com/kubermatic/kubermatic/issues/6634) 
```

Comment:
IIRC, this has been reverted https://github.com/kubermatic/kubermatic/pull/6906/files
cc @zreigz

Reply:
Removed